recipe/meta.yaml

```
@@ -39,15 +39,19 @@ requirements:
     - cyrus-sasl  # [unix]
     - libmagic  # [unix]
     - serf  # [linux]
+    - swig
+    - perl >=5.8.0
```

To use the conda-forge global pinning

```suggestion
    - perl
```